app/routes/events/components/EventDetail/index.js

```diff
@@ -460,21 +476,6 @@ export default class EventDetail extends Component<Props, State> {
               </Flex>
             </ContentSidebar>
           </ContentSection>
-          <Link
-            to="/pages/arrangementer/26-arrangementsregler"
-            style={{ marginTop: 0 }}
-          >
-            <Flex alignItems="center">
-              <Icon name="document" style={{ marginRight: '4px' }} />
-              <span>Regler for Abakus&#39; arrangementer</span>
-            </Flex>
-          </Link>
-          {loggedIn && (
-            <p>
-              Du kan oppdatere dine allergier og preferanser
```

Comment:
> And in addition, with this, we're not consistent in our use of links: with the allergy link, it's at the end of the sentence, whereas with the rules, the link is the the entire sentence. Does that make sense? It looks a bit strange next to each other. I would prefer to not use links in the middle or at the end of sentences on a webapp (e.g. "klikk her for å [...]"), it should in stead be intuitive by means of buttons, icons or other practices.

Very much agree, it looks a bit strange with the inconsistency. I could change them out, but so far I haven't come up with any other clear way to format this than as links - as they either require too much context to fit in a button, and they don't really perform any actions - they are just links..  

I could switch them both out for full-width links though, might be a little bit better. Open for suggestions if you have any other ideas.

Reply:
The allergy link might make sense as a button with a text like "Oppdater allergier og preferanser". It's not really an action, but it is a link to a page where you would perform an action. 

Not sure how it would look with one button and one link though. 

Or maybe if you shortened down the text a bit, made the whole thing a link, and then threw in some food icon, so it matches the rules with the document icon.

Reply:
Good ideas! Updated both to links with icons now, with the same formatting as the time and place. Updated the pics in the description

Didn't find any nice icons with food in ion, so I went with an edit-icon

Reply:
There should be a pizza icon, but the edit-icon also works fine

Reply:
True - tried it out now but as the link also refers to preferences, and they are on the same page, it ended up being clearer that you can edit both than using some variant of pizza and cog.